src/Workspaces/Core/Portable/EmbeddedLanguages/LanguageServices/IEmbeddedLanguage.cs

```diff
@@ -11,9 +11,5 @@ namespace Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices
     /// </summary>
     internal interface IEmbeddedLanguage
```

Comment:
questionable if IEmbeddedLanguage serves any purpose now.  a followup PR will try to remove it.